staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -742,7 +742,7 @@ func (c *RecommendedConfig) Complete() CompletedConfig {
 	return c.Config.Complete(c.SharedInformerFactory)
 }
 
-var allowedMediaTypes = []string{
+var AllowedMediaTypes = []string{
```

Comment:
Global shared access to variables like this is something I'd like to avoid.

Reply:
For this case, is there some way we could make it possible to added the CBOR media type via configuration?